### ISSUE TITLE
Lower the api pod replicas from 3 to 1

### ISSUE THREAD

**Issue by rgl** [kubernetes/dashboard#8812]
### What would you like to be added?

Lower the api pod replicas from 3 to 1.

### Why is this needed?

To lower the requirements.

Also, all the other pods do not have more than one replica. Which makes this pod having more than one, somewhat strange to me. Unless I'm missing something?

**Comment by floreks** [comment]
Our API uses very little resources when idle. Since this is mostly a proxy to Kubernetes API server it's good to balance the load through more pods. It can also help with i.e. throttling on bigger clusters or with more users.

You can override it if you don't need it.

**Comment by rgl** [comment]
and one instance would use even less resources :)

resources do not "idle" per-se. they are actually committed/dedicated as defined in their resource limits settings.

a single instance should handle a substancial load, right? more so, being mostly a proxy?

indeed, I've scaled it down to one. I'm just trying to understand why there is more than one by default.

**Comment by floreks** [comment]
> resources do not "idle" per-se. they are actually committed/dedicated as defined in their resource limits settings.

Of course. That's why most kubernetes clusters are heavily overprovisioned and actual resource consumption is a lot lower than configured limits.

> a single instance should handle a substancial load, right? more so, being mostly a proxy?

Having not done any performance/load tests of our API, I can't say anything for sure. I assume that Kubernetes API will be a bottleneck here since it tends to throttle requests quite heavily on bigger clusters.

It's hard to determine what kind of clusters should we target by default. Certainly small to medium clusters should not have any issues running on a single replica. Let's try scaling it back to 1 for now.
